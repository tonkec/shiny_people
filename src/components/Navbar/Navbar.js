import React from "react";

const Navbar = () => (
  <nav className="nav">
    <div className="container">
      <div className="user container__inner">
        <div className="user__image"></div>
        <div className="user__data">
          <h3 className="user__name">Antonija</h3>
          <span>Admin</span>
        </div>
      </div>
    </div>
  </nav>
);
export default Navbar;
