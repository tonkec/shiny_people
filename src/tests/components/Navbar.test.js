import React from "react";
import Navbar from "components/Navbar";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

describe("Navbar", () => {
  it("should render Navbar correctly", () => {
    const wrapper = shallow(<Navbar />);
    expect(toJSON(wrapper)).toMatchSnapshot();
    expect(wrapper.find(".user__image").length).toBe(1);
  });
});
