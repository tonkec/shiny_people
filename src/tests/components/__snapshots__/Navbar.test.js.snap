// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render Navbar correctly 1`] = `
<nav>
  <div
    className="container"
  >
    <div
      className="user container__inner"
    >
      <div
        className="user__image"
      />
      <div
        className="user__data"
      >
        <h3
          className="user__name"
        >
          Antonija
        </h3>
        <span>
          Admin
        </span>
      </div>
    </div>
  </div>
</nav>
`;
